examples/consumer_example/consumer_example.go

```diff
@@ -81,11 +85,24 @@ func main() {
 
 			switch e := ev.(type) {
 			case *kafka.Message:
+				// Process the message received.
 				fmt.Printf("%% Message on %s:\n%s\n",
 					e.TopicPartition, string(e.Value))
 				if e.Headers != nil {
 					fmt.Printf("%% Headers: %v\n", e.Headers)
 				}
+
+				// We can store the offsets of the messages manually or let
```

Comment:
I think we also need to explain what storing the offset means, because this might give the idea that storing == committing.

Reply:
Done, tried to explain the difference between storing and committing.

Just wanted to confirm if it's okay to use this line or if this is the recommended way of doing things:
"We generally follow a two-phase process, of storing offsets, and then committing them"